superset/connectors/base/models.py

```diff
@@ -27,6 +27,23 @@
 from superset.models.slice import Slice
 from superset.utils import core as utils
 
+METRIC_FORM_DATA_PARAMS = [
+    "metric",
+    "timeseries_limit_metric",
+    "size",
+    "x",
+    "y",
+    "metric_2",
+]
+
+COLUMN_FORM_DATA_PARAMS = [
+    "columns",
+    "all_columns",
+    "all_columns_x",
+    "groupby",
+    "order_by_cols",
+]
```

Comment:
Two that I believe are missing here: `series` and `entity`.

Reply:
added, good catch!